src/screens/CreatePost/elements/Card.js

```diff
@@ -1,52 +1,51 @@
 import React from 'react';
-import { View, StyleSheet, Platform } from 'react-native';
-import { colors } from '../../../utils/colors';
-const Card = props => {
+import {View, StyleSheet, Platform} from 'react-native';
+import {COLORS} from '../../../utils/theme';
+
+const Card = (props) => {
   return (
     // <View style={{ ...styles.card, ...props.style }}></View>
     <View // Parent
       style={{
         // No backgroundColor
-        shadowColor: '#000',
-        shadowOffset: { width: 0, height: 1 },
+        shadowColor: COLORS.black,
+        shadowOffset: {width: 0, height: 1},
         shadowOpacity: 0.5,
         // shadowRadius: 10,
         zIndex: 999,
         // width: widthInput,
-        ...props.style,
-      }}
-    >
+        ...props.style
+      }}>
       <View // Card
         style={{
           flex: 1,
           borderRadius: 10,
           // To round image corners
           overflow: 'hidden',
-          borderColor: '#bdbdbd',
+          borderColor: COLORS.silver,
           borderWidth: 0.5,
           // https://github.com/facebook/react-native/issues/10049#issuecomment-366426897
-          backgroundColor: colors.lightgrey,
+          backgroundColor: COLORS.lightgrey,
           // Android shadow
           elevation: 4,
           paddingHorizontal: 15,
           paddingVertical: 10,
           width: '100%'
-        }}
-      >
+        }}>
         {props.children}
       </View>
     </View>
   );
 };
 const styles = StyleSheet.create({
   card: {
-    shadowOffset: { width: 10, height: 10 },
+    shadowOffset: {width: 10, height: 10},
     shadowColor: 'black',
     shadowOpacity: 1,
     elevation: 3,
     // background color must be set
-    backgroundColor: "#0000", // invisible color
-    zIndex: 999,
+    backgroundColor: COLORS.black, // invisible color
+    zIndex: 999
   }
 });
-export default Card;
\ No newline at end of file
+export default Card;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, but no patch was found that overlapped with it. Original lines [52-52]

There is no newline at the end of the file. While this won't cause any issues with the functionality of your code, it's a common convention to end files with a newline. This can help prevent any potential issues when concatenating or compiling files.
```diff
- export default Card;
+ export default Card;\n
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->